### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by chaceburnette**

### CODE REVIEW

**Review by TomMahle — APPROVED** [review]

File: osmosis/src/setupStore.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ const setupStore = (useCustomHook, config = {}) => {
         <WrappedComponent {...props} />
       </StoreContextWrapper>
     );
+    Wrapper.displayName = `${useCustomHook.name}Wrapper`;
```
Comment (crosenfrisk) [inline osmosis/src/setupStore.js:86]: Should there be a space between the curly brace and the word Wrapper?

File: osmosis/yarn.lock [hunk 2]
```diff
@@ -1391,7 +1391,7 @@ component-emitter@^1.2.1:
 concat-map@0.0.1:
   version "0.0.1"
   resolved "https://registry.yarnpkg.com/concat-map/-/concat-map-0.0.1.tgz#d8a96bd77fd68df7793a73036a3ba0d5405d477b"
-  integrity sha1-2Klr13/Wjfd5OnMDajug1UBdR3s=
+  integrity sha512-/Srv4dswyQNBfohGpz9o6Yb3Gz3SrUDqBH5rTuhGR7ahtlbYKnVxw2bCFMRljaA7EXHaXZ8wsHdodFvbkhKmqg==
```
Comment (crosenfrisk) [inline osmosis/yarn.lock:1394]: Just double-checking that we want two == at the end of line 1394.

File: osmosis/src/setupStore.js (hunk 1 above)
Comment (chaceburnette) [reply osmosis/src/setupStore.js:86]: no, this is intentionally as shown above to ensure the display name does not contain spaces 😉

File: osmosis/yarn.lock (hunk 2 above)
Comment (chaceburnette) [reply osmosis/yarn.lock:1394]: This is actually a generated fingerprint, so its correct as shown 😉

**Review by crosenfrisk — APPROVED** [review]

**Review by joshuakg — APPROVED** [review]

**Pull request merged**